core/src/main/java/com/novoda/noplayer/NoPlayerView.java

```diff
@@ -29,15 +30,16 @@ public NoPlayerView(Context context, AttributeSet attrs) {
     public NoPlayerView(Context context, AttributeSet attrs, int defStyleAttr) {
         super(context, attrs, defStyleAttr);
         surfaceHolderProvider = new PlayerViewSurfaceHolder();
-        aspectRatioChangeListener = new AspectRatioChangeListener(this);
+        aspectRatioChangeListener = new AspectRatioChangeForwarder(this);
     }
 
     @Override
     protected void onFinishInflate() {
         super.onFinishInflate();
         View.inflate(getContext(), R.layout.noplayer_view, this);
-        videoFrame = Views.findById(this, R.id.video_frame);
+        videoFrame = Views.findById(this, R.id.exo_content_frame);
```

Comment:
This the `NoPlayerView` however we are using a layout named `exo_content_frame`. Is that ok?